python/plugins/processing/algs/qgis/SplitLinesWithLines.py

```diff
@@ -2,7 +2,7 @@
 
 """
 ***************************************************************************
-    SplitLines.py
+    DEPRECATED, replaced by SplitWithLines.py
```

Comment:
Processing has a proper way to deprecate algorithms - see https://github.com/qgis/QGIS/blob/master/python/plugins/processing/algs/qgis/PolygonCentroids.py#L51

This will allow the algorithm to still function in existing models, but hides it away in the interface